tests/php/test-amp-style-sanitizer.php

```diff
@@ -982,6 +982,16 @@ public function get_amp_selector_data() {
 				'.main-navigation ul ul li:hover > ul, .main-navigation ul ul li.focus > ul { left: 100%; right: auto; } nav.focused { outline:solid 1px red; }',
 				'.main-navigation ul ul li:hover > ul,.main-navigation ul ul li:focus-within > ul{left:100%;right:auto}nav.focused{outline:solid 1px red}',
 			],
+			'focus_selector_after_whitespace_combinator' => [
+				'<nav class="main-navigation focused"><ul><li class="menu-item-has-children"><a href="https://example.com/">Example</a><ul><li><a href="https://example.org">Another example</a></li></ul></li></ul></nav>',
+				'.main-navigation ul li:hover > ul, .main-navigation ul .focus > ul { left: 100%; right: auto; } nav.focused { outline:solid 1px red; }',
```

Comment:
I found that changing `ul .focus > ul` to `ul     .focus > ul` causes it to fail.

I also found that a selector like `.focus > ul` also fails, when `.focus` is at the beginning.

Reply:
Fixed in 9cf2a14fca44bc7cfa6da9c095cfca1c4f992ca8